pytest.ini

```diff
@@ -1,5 +1,6 @@
 [pytest]
-addopts = -v -x --pyargs foreshadow --cov=foreshadow --xdoc
```

Comment:
We might want to not merge this change in?

Reply:
Will remove my bad